en/identity-server/next/mkdocs.yml

```
@@ -866,6 +866,11 @@ nav:
       - SCIM 2.0 Bulk API: apis/organization-apis/org-scim2-bulk-mgt.md
       - SCIM 2.0 Group management API: apis/organization-apis/org-group-mgt.md
       - SCIM 2.0 Role management API: apis/organization-apis/org-role-mgt.md
+      - Server management:
+          - Email templates:
+              - Email templates v1 API: apis/organization-apis/email-templates-rest-api.md
+              - Email templates v2 API: apis/organization-apis/email-templates-v2-rest-api.md
```

```suggestion
              - Email templates v1 API: apis/organization-apis/email-templates-rest-api.md
              - Email templates v2 API: apis/organization-apis/email-templates-v2-rest-api.md
```

Don't we need to mark these two APIs as (deprecated)

---

Confirm the deprecation status of management API and edit accordingly

---

Since we haven't removed the usages in above APIs fully in console. Added a comment [1] for the docs update  after deprecation.

[1] - https://github.com/wso2/product-is/issues/21948#issuecomment-2644467916